matchbox_socket/src/ggrs_socket.rs

```diff
@@ -1,49 +1,31 @@
-use std::marker::PhantomData;
-
 use ggrs::{Message, PlayerType};
 use matchbox_protocol::PeerId;
 
 use crate::{
-    ChannelConfig, ChannelPlurality, MessageLoopFuture, MultipleChannels, NoChannels, Packet,
-    SingleChannel, WebRtcChannel, WebRtcSocket, WebRtcSocketBuilder,
+    ChannelConfig, MessageLoopFuture, Packet, WebRtcChannel, WebRtcSocket, WebRtcSocketBuilder,
 };
 
+pub const GGRS_CHANNEL_ID: usize = 0;
```

Comment:
Hmm, I think there's a case to be made for making it non-pub entirely (it's only used in this module), but since "ggrs channel must be the first channel added and so will have channel id of 0" is now enforced by the `pub` `add_ggrs_channel()`, the ggrs channel ID effectively already forms part of the public API, and so I think it makes sense to have this constant be `pub`.

That said, I forgot to export the constant, so I should probably do that.